cpp/test/test_info.cc

```diff
@@ -108,6 +110,13 @@ TEST_CASE("test_vertex_info") {
   REQUIRE(v_info.AddPropertyGroup(pg2).IsInvalidOperation());
   REQUIRE(v_info.GetPropertyGroups().size() == 1);
 
+  GAR_NAMESPACE::Property p2;
+  p2.name = "name";
+  p2.type = GAR_NAMESPACE::DataType(GAR_NAMESPACE::Type::STRING);
+  p2.is_primary = false;
+  GAR_NAMESPACE::PropertyGroup pg3({p2}, GAR_NAMESPACE::FileType::CSV);
+  REQUIRE(v_info.AddPropertyGroup(pg3).ok());
+
```

Comment:
Can you add the missing tests (TODO items) in this file? They are already implemented in [construct_info_example.cc](https://github.com/alibaba/GraphAr/blob/main/cpp/examples/construct_info_example.cc)

Reply:
OK, but this is not related to this pull request, I will address the issue with another pull request